addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2178,9 +2178,9 @@ options.registry.Box = options.Class.extend({
         this.$target.toggleClass(params.shadowClass, !!widgetValue);
         const defaultShadow = this._getDefaultShadow(widgetValue, params.shadowClass);
         this.$target[0].style.setProperty('box-shadow', defaultShadow, 'important');
-        if (widgetValue === 'outset') {
-            // In this case, the shadowClass is enough
-            this.$target[0].style.setProperty('box-shadow', '');
+        if (widgetValue === 'outset' || widgetValue === '') {
+            // In this case, the shadowClass is enough for 'outset'
+            this.$target[0].style.setProperty('box-shadow', 'none');
```

Comment:
I don't understand this fix, this code literally says: "if we choose an outset shadow, then add the shadow class, force it as important inline style, then set to it none"... it does not make sense. Forcing to none is indeed what you are trying to do for the "no-shadow" case but for the "outset" case I don't see how it could make sense.

Then even in the "no-shadow" case, this could not be perfect: we don't want that removing the shadow adds a `box-shadow: none` on items which would naturally have no shadow. That increases the DOM size for no reason (e.g. on a normal column, if you add a shadow and remove it... we just want the column back to what it was: no inline style, no shadow). Instead you could try something like we do in the base `selectStyle` method with the `applyCSS` inline function (maybe create an utility function for it if necessary?): apply the style only if it would have an effect.